src/Microsoft.AspNet.Mvc.Xml/DefaultWrapperProviderFactoryProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+    public class DefaultWrapperProviderFactoryProvider : IWrapperProviderFactoryProvider
```

Comment:
`DefaultWrapperProviderFactoryProvider` is an example of a composite. so it would probably be `CompositeWrapperProviderFactoryProvider`.

Regarding the startup code, not sure we have locked on that yet. Either options or a property on the formatter? your suggestion?